test/rules/dot_product/test_in2.jl

```diff
@@ -0,0 +1,32 @@
+module RulesDotProductIn2Test
+
+using Test
+using ReactiveMP
+using Random
+import ReactiveMP: @test_rules
+import LinearAlgebra: dot
+
+@testset "rules:typeof(dot):in2" begin
+
+     @testset "Belief Propagation: (m_out::UnivariateNormalDistributionsFamily, m_in1::PointMass)" begin
```

Comment:
I am not sure whether the `meta` argument is automatically passed to the test function. @bvdmitri could you verify this?